src/group_theory/submonoid/operations.lean

```diff
@@ -388,25 +388,29 @@ end galois_insertion
 
 end submonoid
 
-namespace submonoid_class
+namespace one_mem_class
 
-variables {A : Type*} [set_like A M] [hA : submonoid_class A M] (S' : A)
+variables {A M₁ : Type*} [set_like A M₁] [has_one M₁] [hA : one_mem_class A M₁] (S' : A)
 include hA
 
 /-- A submonoid of a monoid inherits a 1. -/
 @[to_additive "An `add_submonoid` of an `add_monoid` inherits a zero."]
```

Comment:
These docstrings are now a bit misleading

Reply:
I thought about it, but specifically left them like that. We do this kind of mild lying in docstrings all the time.